R/chat.R

```diff
@@ -463,13 +469,20 @@ rlang::on_load(
         session = session
       )
     }
+
     chat_append_message(
       id,
       list(role = role, content = ""),
       chunk = "end",
       operation = "append",
       session = session
     )
+
+    if (every(res, is.character)) {
```

Comment:
Where is `every` coming from? purrr?

Reply:
https://github.com/posit-dev/shinychat/blob/main/R/import-standalone-purrr.R